docs/components/Avatar.tsx

```diff
@@ -8,7 +8,7 @@ export const Avatar = ({ name, picture, twitterUsername }: AuthorDetails) => (
         src={picture}
         height={32}
         width={32}
-        loading="lazy"
+        loading="eager"
```

Comment:
leaving out the attr would be the same too

Reply:
Lazy is actually the default https://nextjs.org/docs/api-reference/next/image#loading

Reply:
(swapped to priority instead)